src/main/views/contact-application.njk

```diff
@@ -1,64 +0,0 @@
-{% extends "template.njk" %}
```

Comment:
Do we really need to delete this contact-application.njk file? May be it can be re-used (copy/paste) for other pages?